examples/make_eeg_bids.py

```diff
@@ -0,0 +1,198 @@
+# Now we just need to specify a few more EEG details to get something sensible:
+
+# First, tell `MNE-BIDS` that it is working with EEG data:
+kind = 'eeg'
+
+# Brief description of the event markers present in the data. This will become
+# the `trial_type` column in our BIDS `events.tsv`. We know about the event
+# meaning from the documentation on PhysioBank
+trial_type = {'rest': 0, 'imagine left fist': 1, 'imagine right fist': 2}
+
+# Now convert our data to be in a new BIDS dataset.
+raw_to_bids(subject_id=subject_id, task=task, raw_file=raw_file,
+            output_path=output_path, kind=kind, event_id=trial_type)
+
+###############################################################################
+# What does our fresh BIDS directory look like?
+print_dir_tree(output_path)
+
+###############################################################################
+# Looks good so far, let's convert the data for all tasks and subjects.
+
+# Start with a clean directory
+sh.rmtree(output_path)
+
+# Some initial information that we found in the PhysioBank documentation
+task_names = {2: 'resteyesclosed',
+              4: 'imaginefists',  # run 1
+              12: 'imaginefists'  # run 2
+              }
+
+run_mapping = {2: None,  # for resting eyes closed task, there was only one run
+               4: '1',
+               12: '2'
+               }
+
+# Now go over subjects and *bidsify*
+for subj_idx in [1, 2]:
+    for task_idx in [2, 4, 12]:
+        # Load the data
+        edf_path = os.path.join(data_dir,
+                                'S{:03}'.format(subj_idx),
+                                'S{:03}R{:02}.edf'.format(subj_idx, task_idx))
+        raw = read_raw_edf(edf_path, preload=True)
+
+        # `kind` and `trial_type` were already defined above
+        raw_to_bids(subject_id='{:03}'.format(subj_idx),
+                    task=task_names[task_idx],
+                    run=run_mapping[task_idx],
+                    raw_file=raw,
+                    output_path=output_path,
+                    kind=kind,
+                    event_id=trial_type,
+                    overwrite=False
+                    )
+
+###############################################################################
+# Step 3: Check and compare with standard
+# ---------------------------------------
+# Now we have written our BIDS directory.
+print_dir_tree(output_path)
+
+###############################################################################
+# MNE-BIDS has created a suitable directory structure for us, and among other
+# meta data files, it started an `events.tsv` and `channels.tsv` and made an
+# initial `dataset_description` on top!
+#
+# That's nice and it has saved us a lot of work. However, a few things are not
+# yet covered by MNE-BIDS for EEG data and will require some manual tweaking.
+# Make sure to pay a lot of attention to the automatic output and make
+# adjustments where necessary.
```

Comment:
Instead of putting a disclaimer, I would suggest pointing the user to look for `'n/a'` entries in the data to fill them up themselves. Is there actually a possibility of MNE-BIDS putting any information incorrectly? We don't do any guessing, right?

Reply:
> We don't do any guessing, right?

We don't and we shouldn't - I'll rephrase!